ui/pages/settings/networks-tab/networks-form/networks-form.js

```diff
@@ -446,19 +465,27 @@ const NetworksForm = ({
           (chain) => chain.chainId.toString() === decimalChainId,
         );
 
+        const matchedSymbols = safeChainsList.current?.reduce(
```

Comment:
It looks like this code/logic gets repeated -- can we create a helper function for this?